.changeset/fifty-experts-exist.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': patch
```

Comment:
No need for a changeset here actually, the sample templates aren't part of the release 😅

Reply:
Good to know, I will remove it.